src/plugins/requestAuthPlugin.js

```diff
@@ -107,17 +107,17 @@ export const applyRequestAuthorizer$ = requestAuthorizer$ => request => {
  * Get an anonymous code from the API that can be used to generate an oauth
  * access token
  *
- * @param {Object} config { OAUTH_AUTH_URL, oauth }
+ * @param {Object} server the Hapi app server instance
```

Comment:
Interestingly, `server.settings.app` appears to be the key to fixing the integration tests.

When the plugin `register` function is called, it doesn't appear to receive `server.app`, but it does receive `server.settings.app`. If I squint, I can sort of see the reasoning behind this since `server.app` is intended to be for runtime state, and plugins are meant to be loaded statically, so should in theory only be provided with static configuration info.

I'll start refactoring to `server.settings.app` and confirm that it is working as intended